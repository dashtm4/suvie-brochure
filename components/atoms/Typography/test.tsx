import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Typography } from '.'

describe('<Typography />', () => {
  it('should renders correct <Typography />', () => {
    const wrapper = shallow(<Typography />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
