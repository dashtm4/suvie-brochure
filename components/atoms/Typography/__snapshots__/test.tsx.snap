// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Typography /> should renders correct <Typography /> 1`] = `
<div
  className="text-secondary"
/>
`;
